changelog.txt

```diff
@@ -15,6 +15,7 @@
 * Add - Introduced deposit currency filter for deposits overview page.
 * Update - Make Payment Request Button available for all merchants.
 * Add - Configurable Payment Request Button locations.
+* Add - Support saved payments for SEPA.
```

Comment:
Let's omit the changelog entry here, we'll add a new one when releasing the payment method.